src/Plugin/MediaEntity/Type/Twitter.php

```diff
@@ -163,6 +176,20 @@ public function getField(MediaInterface $media, $name) {
             return $tweet['retweet_count'];
           }
           return FALSE;
+
+        case 'thumbnail':
+          $uri = 'public://twitter-thumbnails/' . $matches['id'] . '.svg';
+          if (! file_exists($uri)) {
+            $build = [
+              '#theme' => 'media_entity_twitter_tweet_thumbnail',
+              '#tweet' => $tweet['text'],
+              '#author' => $tweet['user']['name'],
+              '#avatar' => $tweet['user']['profile_image_url'],
```

Comment:
I don't have a good answer for this off the top of my head. It'd be expensive and extremely difficult (potentially impossible) to regenerate the thumbnails entirely every time they're viewed. So for the time being, I don't have any solution to this problem yet. (The username is embedded in the thumbnail, at least, and that won't change.)